node/pkg/adminrpc/adminserver.go

```diff
@@ -33,10 +35,18 @@ import (
 	"github.com/wormhole-foundation/wormhole/sdk/vaa"
 )
 
+var (
+	vaaInjectionsTotal = promauto.NewCounter(
+		prometheus.CounterOpts{
+			Name: "wormhole_vaa_injections_total",
+			Help: "Total number of injected VAA queued for broadcast",
+		})
+)
+
```

Comment:
It might be cool to add labels for this for the chain these are injected to.

Reply:
feel free :)